assignments/assignment-8/assignment-8.7.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8.7</title>
+    </head>
+    <body>
+        <h2>Photo contect application form</h2>
+
+        <label for="fullNameInput">Full Name</label>
+        <input
+            type="text"
+            placeholder="Full Name"
+            id="fullNameInput"
+            name="fullName"
+            required
+        />
+        <label for="emailInput">Email</label>
+        <input
+            type="email"
+            placeholder="Email"
+            id="emailInput"
+            name="email"
+        /><br /><br />
```

Comment:
It is better to use div instead of br in this case. Like:
"div" label input "/div"

Reply:
yes but it doesn't go down accurately, but yeah I will try

Reply:
It is not mistake but it is recommended  as I had done the same